sagemaker-featurestore/feature_store_kms_key_encryption.ipynb

```diff
@@ -0,0 +1,560 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Amazon SageMaker Feature Store: Encrypt Data in your Online or Offline Feature Store using KMS key"
```

Comment:
"Amazon SageMaker Feature Store: Encrypt data in your online or offline feature store using a KMS key"